e2e-polybft/e2e/jsonrpc_test.go

```diff
@@ -26,6 +26,7 @@ func TestE2E_JsonRPC(t *testing.T) {
 
 	cluster := framework.NewTestCluster(t, 4,
 		framework.WithPremine(types.Address(acct.Address())),
+		// framework.WithHTTPS("/etc/ssl/certs/ssl-cert-snakeoil.pem", "/etc/ssl/private/ssl-cert-snakeoil.key"),
```

Comment:
I would like to have a CLI command that would create the SSL certificate and some E2E test that would test it is working without problems via https as well. However, it can be a separate PR. For this PR, we can remove this line. WDYT?

Reply:
This is in accordance with today's agreement with Nemanja. Existing tests
will be moved to https once CI infrastructure is ready for that.